model/src/channel/message/interaction.rs

```diff
@@ -6,6 +6,7 @@ use crate::{
 };
 use serde::{Deserialize, Serialize};
 
+/// Response to an interaction without an existing message.
```

Comment:
I don't fully understand message interactions and it's very poorly documented, so I'd appreciate if someone with experience could add some more lines on when it's received and how to handle it.